server/plugin/virtual_agent_test.go

```diff
@@ -212,3 +216,82 @@ func Test_CreatePickerAttachment(t *testing.T) {
 		})
 	}
 }
+
+func Test_CreateMessageAttachment(t *testing.T) {
+	p := Plugin{}
+
+	defer monkey.UnpatchAll()
+
+	for _, testCase := range []struct {
+		description      string
+		fileID           string
+		response         *MessageAttachment
+		getFileInfoError *model.AppError
+		marshalError     error
+		encryptError     error
+		expectedError    string
+	}{
+		{
+			description: "CreateMessageAttachment returns valid attachment",
+			fileID:      "mockFileID",
+			response: &MessageAttachment{
+				URL:         "mockSiteURL" + p.GetPluginURLPath() + "/file/" + encode([]byte{}),
+				ContentType: "mockMimeType",
+				FileName:    "mockName",
+			},
+		},
+		{
+			description: "CreateMessageAttachment returns error while getting file info",
```

Comment:
Check this every where
```suggestion
			description: "CreateMessageAttachment returns an error while getting file info",
```